cypress/integration/page.spec.js

```diff
@@ -556,26 +585,26 @@
   describe('Posts List', () => {
     beforeEach(() => {
       page.mockUsers();
-      page.mockUser1Posts()
+      page.mockUser1Posts();
```

Comment:
The `page.mockUser1Posts()` call should be followed by `.as('user1PostsRequest')`.